pynest/nest/lib/hl_api_types.py

```diff
@@ -442,16 +450,27 @@ def index(self, node_id):
             If the node ID is not in the `NodeCollection`.
         """
         index = sli_func('Find', self._datum, node_id)
+
         if index == -1:
             raise ValueError('{} is not in NodeCollection'.format(node_id))
+
         return index
 
+    def __array__(self, dtype=None):
+        ''' Numpy compatibility function '''
```

Comment:
The docstring should describe what the function does and it should be in triple double quotes, e.g.:
```suggestion
        """Convert the NodeCollection to a NumPy array."""
```